package.json

```diff
@@ -1,8 +1,20 @@
 {
+  "engines": {
+    "node": ">=17.0.0"
+  },
   "dependencies": {
     "@actions/core": "^1.2.6",
     "@cfworker/json-schema": "^1.8.2",
     "semver": "^7.3.4"
   },
-  "type": "module"
+  "type": "module",
+  "devDependencies": {
+    "@babel/eslint-parser": "^7.17.0",
```

Comment:
Do we even need babel, for this to work? We're on a pretty new version of JS :thinking: 

Non-blocker, just a question.

Reply:
yes, because the eslint default parser was not able to infer most of the code. I've tested that.

Reply:
Oh interesting, I have changed the config lately and maybe thats why babel is not required anymore.
I'll drop it as `eslint` actually also works without it. 👍🏼